.github/workflows/test.yaml

```
@@ -74,10 +82,11 @@ jobs:
           "solrcloud-upload-configset.sh /app/samvera/hyrax-webapp/solr/config &&
           SOLR_COLLECTION_NAME=hydra-test solrcloud-assign-configset.sh &&
           solrcloud-assign-configset.sh"
+        if: ${{ inputs.solr }}
```

Kiah - had some trouble getting the suggestion to go high enough, so you cant just accept this, but instead of doing the whole if thing, you could make the Setup solr only run if a "solr" directory is present. The present of solr directory then causes solr stuff to happen.

```suggestion
         [ -f solr ] &&
         docker-compose exec -T web bash -c
          "solrcloud-upload-configset.sh /app/samvera/hyrax-webapp/solr/config &&
          SOLR_COLLECTION_NAME=hydra-test solrcloud-assign-configset.sh &&
          solrcloud-assign-configset.sh"
        if: ${{ inputs.solr }}
```